ctx_test.go

```diff
@@ -3107,6 +3176,23 @@ func (t *testTemplateEngine) Load() error {
 	return nil
 }
 
+type testTemplateEngine2 struct {
```

Comment:
Instead of copying them, please make the path in testTemplateEngine configurable